examples/desktop/main.go

```diff
@@ -1,18 +1,20 @@
 /*
- * Copyright (C) 2018 The "MysteriumNetwork/go-openvpn" Authors.
+ * go-openvpn -- Go gettable library for wrapping Openvpn functionality in go way.
+ *
+ * Copyright (C) 2020 The "MysteriumNetwork/go-openvpn" Authors..
```

Comment:
Here should be Copyright (C) 2020 Blockdev AG

Reply:
Fixed